example/posts.js

```diff
@@ -44,7 +44,7 @@ import {
     required,
     translate,
 } from 'admin-on-rest'; // eslint-disable-line import/no-unresolved
-import RichTextInput from 'aor-rich-text-input';
+// import RichTextInput from 'aor-rich-text-input';
```

Comment:
Did you forget to uncomment ?

Reply:
No, see the PR description: If I leave it uncommented, the example breaks because the `RichRextInput` hasn't been updated to use the `addField` HOC.

Reply:
Oh sorry about that